gravitee-am-service/src/main/java/io/gravitee/am/service/impl/DomainServiceImpl.java

```diff
@@ -615,6 +618,20 @@ private Completable validateDomain(Domain domain) {
             }
         }
 
+        if (domain.getWebAuthnSettings() != null) {
+            final String origin = domain.getWebAuthnSettings().getOrigin();
+            if (origin == null || origin.isBlank()) {
+                return Completable.error(new InvalidWebAuthnConfigurationException("Error: Invalid origin. Please provide a valid origin."));
+            }
+
+            final URI uri = UriBuilder.fromURIString(origin).build();
+            final List<String> schemes = Arrays.asList("http", "https");
```

Comment:
Moving this array into a constant attribute maybe better than creating an instance each time. But as it is only on Management (so low traffic), I will approve it :)